app/views/items/_file.html.erb

```diff
@@ -0,0 +1,32 @@
+<div class="container"><%# Need additional container here %>
+  <div class="row">
+  Workspace:
+<% if @available_in_workspace &&  %>
```

Comment:
is this a trailing `&&` with nothing on the other side, or am i misreading it?  if so, typo, or is there a rationale?